tox.ini

```diff
@@ -68,3 +68,31 @@ commands =
         --cache-dir="{toxworkdir}/mypy_cache"  \
         {tty:--pretty:}                        \
         {posargs:pydoctor}
+
+
+[testenv:apidocs]
+description = Build only the API documentation
+
+whitelist_externals =
+    cat
+    test
+commands =
+    pydoctor \
+    --add-package=pydoctor \
+    --project-name=pydoctor \
+    --project-url=https://github.com/twisted/pydoctor/ \
+    --html-viewsource-base=https://github.com/twisted/pydoctor/tree/master \
+    --html-output=build/apidocs \
+    --project-base-dir="{toxinidir}" \
```

Comment:
Perhaps it is a good moment to move our source code to a `src` directory (#163), now that there are few open PRs and before making more changes to our build.

Reply:
I can do it when there are not many PR.

I have a few PR that I would like to merge and then we can move to src.

For pydoctor, I don't see as a critical task... I assume that people are using pydoctor for development and for non-critical environments.

I am full supported of automated tests for anything that is production and customer facing/interaction ... but for development tools I prefer a  more Carpe Diem / YOLO approach. 

But we can move to src as it shouldn't be an issue.

Reply:
It's not so much for the users but for ourselves: it's easy to accidentally test the source tree when you think you're testing in a tox env, which can lead to incorrect conclusions and wasting a lot of time.

Another reason is a bit more selfish: since many projects are now using `src`, if I'm searching in the source I'll type `grep -r something src` automatically and then have to correct the directory name.

Reply:
It doesn't need to happen right now, but the sooner we do it, the fewer non-source files we need to update, so it would be nice to do it before releasing.